packages/dumb/src/components/trends/common/trends-svg-container/trends-svg-container.tsx

```diff
@@ -0,0 +1,210 @@
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { type FunctionComponent, useMemo } from 'react'
+import sizeMe from 'react-sizeme'
+import { scaleLinear } from 'd3-scale'
+import { TimeService } from 'medical-domain'
+
+import { type RangeSegmentSlice } from '../../../../models/enums/range-segment.enum'
+import { NoDataLabel } from '../no-data-label/no-data-label'
+import { CbgDateTracesAnimationContainer } from '../../cbg/cbg-date-trace/cbg-date-traces-animation-container'
+import { type CbgDateTrace } from '../../../../models/cbg-date-trace.model'
+import { THREE_HRS } from '../../../../utils/datetime/datetime.util'
+import { getCbgsIntersectingWithCbgSliceSegment } from './trends-svg-container.util'
+import { type BgPrefs } from '../../../../models/blood-glucose.model'
+import { Background } from '../background/background'
+import { XAxisLabels } from '../x-axis-labels/x-axis-labels'
+import { XAxisTicks } from '../x-axis-ticks/x-axis-ticks'
+import { YAxisLabelsAndTicks } from '../y-axis-labels-and-ticks/y-axis-labels-and-ticks'
+import { CbgSlicesContainerMemoized as CbgSlicesContainer } from '../../cbg/cbg-slice/cbg-slices-container'
+import { FocusedCbgSliceSegmentMemoized as FocusedCbgSliceSegment } from '../../cbg/cbg-slice/focused-cbg-slice-segment'
+import { TargetRangeLines } from '../target-range-lines/target-range-lines'
+import { type FocusedSlice } from '../../../../models/focused-slice.model'
+
+const BUMPERS = {
+  top: 50,
+  bottom: 30
+}
+
+const MARGINS = {
+  top: 30,
+  right: 10,
+  bottom: 10,
+  left: 40
+}
+
+const SMBG_OPTS = {
+  maxR: 7.5,
+  r: 6
+}
+
+const CHART_WIDTH_M_SIZE = 70
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏This could be renamed to `CHART_WIDTH`